docker/Dockerfile

```diff
@@ -43,14 +43,10 @@ WORKDIR /tmp
 
 RUN git clone https://github.com/scala-native/scala-native.git
 
-WORKDIR /tmp/scala-native/submodules
-
-RUN rm -rf scala && git clone https://github.com/scala/scala.git
-
```

Comment:
Could I work on this?
How do you prefer to target this feature? sbt settings, environment checks, use of external tools (i.e.  cmake) or whatever?